uncontrast_studies/management/commands/process_uncon_row.py

```diff
@@ -0,0 +1,121 @@
+from collections import namedtuple
+
+from django.core.exceptions import ObjectDoesNotExist
+
+from contrast_api.choices import ExperimentTypeChoices
+from contrast_api.data_migration_functionality.studies_parsing_helpers import ProblemInStudyExistingDataException
+from studies.models import Study
+from uncontrast_studies.models import UnConExperiment, UnConSpecificParadigm, UnConTask, UnConSuppressedStimulus, \
+    UnConTargetStimulus, UnConSuppressionMethodType, UnConSuppressionMethodSubType, UnConSuppressionMethod, UnConSample, \
+    UnConProcessingMainDomain, UnConProcessingDomain, UnConFinding, UnConsciousnessMeasurePhase, \
+    UnConsciousnessMeasureType, UnConsciousnessMeasureSubType, UnConsciousnessMeasure, UnConTaskType
+from uncontrast_studies.parsers.consciousness_measure_parser import resolve_consciousness_measures
+from uncontrast_studies.parsers.uncon_data_parsers import resolve_uncon_paradigm, resolve_uncon_task, resolve_uncon_processing_domains
+from uncontrast_studies.parsers.stimulus_parser import resolve_uncon_stimuli, resolve_uncon_stimuli_metadata
+from uncontrast_studies.parsers.suppression_method_parser import resolve_uncon_suppression_method
+from uncontrast_studies.parsers.sample_parser import resolve_uncon_samples
+from uncontrast_studies.parsers.finding_parser import resolve_uncon_findings
+
+ExperimentDuplicate = namedtuple("ExperimentDuplicateTuple", ["key", "id"])
+
+
+def create_uncon_experiment(item: dict, index):
+    try:
+        if item["Paper.DOI"]:
```

Comment:
why both options? also, the right way to check if something exists in a dict is with `in` , or get and default to null, this would just throw an exception and would never get to the next one